### ISSUE TITLE
Incorrectly exported translate function

### ISSUE THREAD

**Issue by rubensworks** [joachimvh/SPARQLAlgebra.js#10]
With the latest version, I'm getting these errors if I want to use the translate function.

It looks like this is being caused by the addition of the `default` keyword on the translate function, not sure if that was intentional. Removing that fixes it for me.

```
node_modules/sparqlalgebrajs/index.d.ts(1,10): error TS2305: Module '"/Users/rtaelman/Documents/UGent/nosync/comunica/packages/actor-sparql-parse-algebra/node_modules/sparqlalgebrajs/lib/sparqlAlgebra"' has no exported member 'translate'.
```

**Comment by joachimvh** [comment]
A wrong `index.d.ts` file got pushed into the package due to `index.ts` not being included in the `tsconfig.json`. I'll push a fix.

**Comment by joachimvh** [comment]
Fixed in dcbf1410b93019ee7a64c2b4d6463b8517ec73af (and new version was published)
